datafusion/tests/path_partition.rs

```diff
@@ -271,7 +274,7 @@ async fn parquet_overlapping_columns() -> Result<()> {
     Ok(())
 }
 
-fn register_partitioned_aggregate_csv(
+async fn register_partitioned_aggregate_csv(
```

Comment:
This doesn't need to be `async` anymore does it?

Reply:
You are right. ive fixed.